src/lib/poracleMessage/commands/script.js

```diff
@@ -319,21 +319,21 @@ exports.run = async (client, msg, args, options) => {
 		}
 
 		if (!message.length) {
-			return await msg.reply('The script specified is empty')
+			return await msg.reply(client.translator.translate('The script specified is empty'))
```

Comment:
translator.translate(

client.translator.translate gives default language, translator.translate is user language

Reply:
is changed

Reply:
well, that one is -- but there are others :-)

Reply:
ALL fixed now! Sorry by the way, try to be more accurate in the future....